13-loops/exercises/08-lucky-number-exercises/01-first-turn-winner/main.go

```diff
@@ -26,4 +26,52 @@ package main
 // ---------------------------------------------------------
 
 func main() {
+	const (
+		maxTurns = 5 // less is more difficult
+		usage    = `Welcome to the Lucky Number Game! 🍀
+
+The program will pick %d random numbers.
+Your mission is to guess one of those numbers.
+
+The greater your number is, harder it gets.
+
+Wanna play?
+`
+	)
+
+	rand.Seed(time.Now().UnixNano())
+
+	args := os.Args[1:]
+
+	if len(args) != 1 {
+		// fmt.Println("Pick a number.")
+		fmt.Printf(usage, maxTurns)
+		return
+	}
+
+	guess, err := strconv.Atoi(args[0])
+	if err != nil {
+		fmt.Println("Not a number.")
+		return
+	}
+
+	if guess < 0 {
+		fmt.Println("Please pick a positive number.")
+		return
+	}
+
+	for turn := 0; turn < maxTurns; turn++ {
```

Comment:
try to use small variable names like t instead-of turn

Reply:
Fixed!